src/biopack/psg/CompSens.h

```diff
@@ -138,6 +140,7 @@ char *opt;        // reserved for weighted sampling modes
     sprintf(fname,"%s/sampling.sch", curexp);
     if((stype[0] =='a') || (stype[0] == 'y'))
     {
+      int ret __attribute__((unused));
```

Comment:
These silence the compiler warning, but dont really address the issue that the compiler warning is meant to alert us to.  An equivalent option would be disabling the compiler warning until we can test things thoroughly enough to know whether code changes (detecting and handling failures) is going to break anything upstream from this.

Reply:
Also agreed. However, I did not want to use -Wno-... options to silence the warnings. These files are compiled when a user compiles their own pulse sequence (at least solids, imaging, and bio sequences) and I did not want them to miss problems with their code. But I also did not want them to be distracted by problems in these "include" files. I figured we could always grep for __attribute__((unused)) to identify where real fixes should be made.

Reply:
Ah, yeah, that makes total sense.

Reply:
btw, I wrote a little log parser to tally warning counts from a log file: https://github.com/tesch1/OpenVnmrJ/blob/fix-test/scripts/grok_warnings.py